test/unit/filters/realtiveTimeFilterSpec.js

```diff
@@ -0,0 +1,60 @@
+/* global describe, it, inject, expect, beforeEach, jasmine, tsNow*/
+
+describe('relativeTime filter', function () {
+  var $filter, _, relativeTimeFilter, minuteSpy, hourSpy
```

Comment:
I've learned, that you can share variables by assigning them to `this`.

See above for explanation.